mediaflow_proxy/extractors/vavoo.py

```
@@ -0,0 +1,147 @@
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)